easybuild/easyconfigs/f/flatbuffers/flatbuffers-2.0.7-GCCcore-11.3.0.eb

```diff
@@ -0,0 +1,57 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeNinja'
+
+name = 'flatbuffers'
+version = '2.0.7'
+
+homepage = 'https://github.com/google/flatbuffers/'
+description = """FlatBuffers: Memory Efficient Serialization Library"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/google/flatbuffers/archive/v%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['4c7986174dc3941220bf14feaacaad409c3e1526d9ad7f490366fede9a6f43fa']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('Ninja', '1.10.2'),
+]
+
+dependencies = [
+    ('Python', '3.10.4'),
+]
+
+# Install into the same dir as the Python extension which is /lib
+configopts = '-DFLATBUFFERS_ENABLE_PCH=ON -DCMAKE_INSTALL_LIBDIR=lib'
+
+exts_defaultclass = 'PythonPackage'
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'download_dep_fail': True,
+    'use_pip': True,
+    'sanity_pip_check': True,
+}
+
+exts_list = [
+    ('flatbuffers', version, {
+        'sources': {
+            'download_filename': SOURCE_TAR_GZ,
+            'filename': '%(name)s-python.tar.gz',
+        },
+        'checksums': ['0ae7d69c5b82bf41962ca5fde9cc43033bc9501311d975fd5a25e8a7d29c1245'],
+    }),
+]
+
+modextrapaths = {'PYTHONPATH': 'lib/python%(pyshortver)s/site-packages'}
+
+sanity_check_paths = {
+    'files': ['include/flatbuffers/flatbuffers.h', 'bin/flatc', 'lib/libflatbuffers.a'],
+    'dirs': ['lib/cmake'],
```

Comment:
also check for `'lib/python%(pyshortver)s/site-packages'` here

Reply:
I thought that gets checked by the sanity check of PythonPackage? But OK, better be explicit

Reply:
The default `sanity_check_paths` used by `PythonPackage` (only) includes `lib/python%(pyshortver)s/site-packages`, that's right.
But if you specify `sanity_check_paths`, you're overriding the default `sanity_check_paths`, and so you should include `lib/python%(pyshortver)s/site-packages`